### PR TITLE
Fix #1 Allow echo not to display a newline

### LINKED ISSUE #1: Allow echo not to display a newline
we should add an option to not display a newline after invoking `echo`

### PR DESCRIPTION
**Pull request by chatelgu**
Hi Maoutdu,

I fixed the pending issue for echo.

I hope will find my marvellous code nice and you will merge it soon. 
Can't wait to be able to use '-n' option in echo. 

BR,
Chatelgu

### CODE REVIEW

**Comment by chatelgu** [comment]
Je teste le scénario. A toi de trouver 2 remarques à faire sur ma pull request... Je te le laisse comme devoir a la maison pour ce weekend.

File: echo.c [hunk 1]
```diff
@@ -1,9 +1,15 @@
 #include <stdio.h>
 
 int main (int argc, char *argv[]) {
+	char endchar='\n';
+	if (!strcmp(argv[1],"-n")) {
```
Comment (maoutdu) [inline echo.c]: Please document the very likely segfault here, if echo is invoked with no argument

File: echo.c [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
+		--argc;
+		++argv;
+		endchar=' ';
+	}
 	while (--argc) {
 		fputs(*++argv, stdout);
-		putchar (argc == 1 ? '\n' : ' ');
+		putchar (argc == 1 ? endchar : ' ');
```
Comment (maoutdu) [inline echo.c]: This will add an extra space at the end if echo is invoked with `-n`, please document it in README.md

**Comment by maoutdu** [comment]
Dear @chatelgu, thank you for your interest ins improving maoutdux. The patch you submitted introduces undocumented behaviours. Please document them before I can merge this pull request.

File: echo.c (hunk 2 above)
Comment (chatelgu) [inline echo.c]: endchar=0 fix the trainling space

**Pull request closed without merging**